components/editor/editor/extensions/index.tsx

```diff
@@ -156,3 +185,8 @@ export const TiptapExtensions = [
     allowFullscreen: true,
   }),
 ];
+
+function highlightCode(code, language) {
+  const result = lowlight.highlight(language, code);
+  return `<pre><code class="hljs ${language}">${result}</code></pre>`;
```

Comment:
Do we have to use this hljs class, can we copy our styles from our current code block style? 

I am happy to fix styles at a later date in any case.

@NiallJoeMaher